utils/src/ooinstall/openshift_ansible.py

```diff
@@ -11,30 +11,46 @@
 
 CFG = None
 
+ROLES_TO_GROUPS_MAP = {
+    'master': 'masters',
+    'node': 'nodes',
+    'storage': 'nfs'
```

Comment:
It depends on our approach. It would be `'master_lb': 'master_lb'`. This is mostly here to handle the pluralization that the groups expected.